src/pageContent/Profile/styles.tsx

```diff
@@ -0,0 +1,8 @@
+export const EmailUsSection = `min-h-screen max-h-full w-screen flex flex-row justify-center items-center py-8 px-3`;
+
+export const HeaderText = `font-bold text-4xl lg:text-3xl text-gray-900 text-center`;
+export const BodyText = `text-xl lg:text-md text-gray-900 text-center`;
+
+export const MyPicture = `mt-10 h-full w-full flex items-end`;
+
+export const MyDescription = `grid grid-cols-8 grid-rows-1 gap-6 h-screen-3/4`;
```

Comment:
By the way flex is always preferred compared to grid unless pretty obvious its like grid style (gallery or sth), but profile not exactly like a grid

Reply:
also pretty weird you put one grid here, and another grid on the index.tsx, as if somehow the grid appears out of nowhere there, just a bit confusing

Reply:
done